evolver/controller/standard/chemostat.py

```diff
@@ -16,51 +20,74 @@ class Config(VialConfigBaseModel):
         flow_rate: float = Field(0, description="Flow rate for dilutions")
         stir_rate: float = Field(8, description="Stir rate")
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self,
```

Comment:
Your points on duplication and error prone-ness are totally valid, however, as we've discussed before, I don't think they outweigh the advantage of sticking with the canonical python pattern of stating init params and assigning them in the init. This is the core aspect of this PR that we had already discussed and concluded, in fact, it was you that weren't in favor of the introspection of the config as you've suggested here. For the record, I equally wasn't a fan when I suggested it as an alternative.

FYI it's totally ok to have a deferred signature for init, this is common when the exact sig is unknown at some interface layer, however, at some point and in someplace, the sig _will_ be explicitly and concretely stated. What you're stating, however, is that it will never be, by design - that when a cls has a ``Config``, it's not allowed to have a init func sig. I just want to disambiguate the two patterns.

>(in the case of this one, we only wanted for the time and buffer, which should eventually have other sources).

This isn't true though, it wanted everything stated precisely for the reason that it's stated in the config - if it didn't need them, they shouldn't be in the config.

The point of this PR is to use the pattern implemented in #37 which is to:

 - explicitly not ref everything by ``self.config.attr`` but instead directly by ``self.attr``
 - use the normal and expected python pattern of both stating and passing in params as stated in the init sig
 - use a semi-factory-ish pattern ``cls.create(config)`` to instead instantiate from a config. (Not really a factory pattern as it always returns an instance of ``cls``, even for different ``classinfo`` values when a descriptor is passed in, i.e., ``cls.create(config): return config.classinfo.create(config.config)`` would make it a factory pattern.)

Reply:
For the record, it was this very topic that prompted me to bring up the discussion point that, if we wanted creation to hinge on a pedantic model, that perhaps we're using it, as we are, incorrectly, and that instead, everything should just _be_ a model.

We decided against this as there are entities needed for device control that are not conducive of pydantic serialization, and their framework for excluding fields seemed a harder and more complicated imposition on the end user than what was outlined in #37.